examples/native_ext/.bazelrc

```diff
@@ -0,0 +1,6 @@
+# Not ready for the MODULE.lock file yet, as of Bazel 7.0.0 there are still some stability issues.
+common --lockfile_mode=off
```

Comment:
nit: it's better to gitignore it instead. You can get some perf benefits locally by Bazel reading the lockfile from your previous work, and not share it to others